doc/source/getting_started/basics.rst

```diff
@@ -1937,21 +1937,39 @@ See :ref:`extending.extension-types` for how to write your own extension that
++==================+===========================+====================+===============================+=========================================+===============================+
+|tz-aware datetime | :class:`DatetimeTZDtype`  | :class:`Timestamp` | :class:`arrays.DatetimeArray` | ``'datetime64[ns, <tz>]'``              | :ref:`timeseries.timezone`    |
++------------------+---------------------------+--------------------+-------------------------------+-----------------------------------------+-------------------------------+
+|Categorical       | :class:`CategoricalDtype` | (none)             | :class:`Categorical`          | ``'category'``                          | :ref:`categorical`            |
++------------------+---------------------------+--------------------+-------------------------------+-----------------------------------------+-------------------------------+
+|period            | :class:`PeriodDtype`      | :class:`Period`    | :class:`arrays.PeriodArray`   | ``'period[<freq>]'``,                   | :ref:`timeseries.periods`     |
+|(time spans)      |                           |                    |                               | ``'Period[<freq>]'``                    |                               |
++------------------+---------------------------+--------------------+-------------------------------+-----------------------------------------+-------------------------------+
+|sparse            | :class:`SparseDtype`      | (none)             | :class:`arrays.SparseArray`   | ``'Sparse'``, ``'Sparse[int]'``,        | :ref:`sparse`                 |
+|                  |                           |                    |                               | ``'Sparse[int32, 0]'``,                 |                               |
+|                  |                           |                    |                               | ``'Sparse[int64, 0]'``,                 |                               | 
+|                  |                           |                    |                               | ``'Sparse[float32, nan]'``,             |                               |
+|                  |                           |                    |                               | ``'Sparse[float64, nan]'``              |                               |
++------------------+---------------------------+--------------------+-------------------------------+-----------------------------------------+-------------------------------+
+| intervals        | :class:`IntervalDtype`    | :class:`Interval`  | :class:`arrays.IntervalArray` | ``'interval'``, ``'Interval'``,         | :ref:`advanced.intervalindex` |
+|                  |                           |                    |                               | ``'Interval[<np.numeric>]'``,           |                               |
```

Comment:
I'm not sure what `np.numeric` is. Can we just have something like `<dtype>` here?

Reply:
`<dtype>` won't work as it's too generic.  What does work is any numpy numeric type, but not numpy `object`.  I'll use `numpy_dtype`